Code/GraphMol/MolDraw2D/catch_tests.cpp

```diff
@@ -103,10 +103,10 @@ const std::map<std::string, std::hash_result_t> SVG_HASHES = {
     {"testBrackets-2b.svg", 776446765U},
     {"testBrackets-2c.svg", 1050851904U},
     {"testBrackets-2d.svg", 776446765U},
-    {"testBrackets-3a.svg", 1374944956U},
+    {"testBrackets-3a.svg", 3096284602U},
```

Comment:
Did you look at the test drawings that changed to make sure the new drawings make sense?

Reply:
I looked at 3a. IIRC the OH had moved.  I don’t recall the other 2. I will check tomorrow morning.

Reply:
I think everything is ok.  A couple, such as testHandDrawn-5[ab].svg are improved.

Reply:
> I think everything is ok. A couple, such as testHandDrawn-5[ab].svg are improved.

Thanks for checking!